google/model-viewer
# Update babylonjs and its goldens

Did four things in this PR:
1.  update babylonjs to latest version
2. remove 0 radius check  and some babylon scenarios from exclude
3. enable real time filtering for some scenarios 
4. update related goldens

Here's how those high dynamic scenarios looks like now:
![image](https://user-images.githubusercontent.com/44676379/88720371-c2f94300-d0f2-11ea-917f-0379e0d93f7d.png)

In older babylonjs:
![image](https://user-images.githubusercontent.com/44676379/88721319-449da080-d0f4-11ea-9f45-fa0d7a9ce68d.png)
